### ISSUE TITLE
slither segmentation fault

### ISSUE THREAD

**Issue by ascbot** [DOI-USGS/ISIS3#2869]
---

Author Name: **Guy McArthur** (Guy McArthur)

---

Slither in ISIS 3.6.1 (Linux x86_64) segfaults on some input control net files (created by hijitreg).

**Comment by jessemapel** [comment]
@all We need more information for this. Should we close it and then reach out to the reporter? Do we have an e-mail from redmine?

**Comment by jlaura** [comment]
@jessemapel I pinged the reporter via email.

**Comment by guymac** [comment]
Yes, this is still an issue. We are stuck at ISIS 3.5 until this can be resolved.

**Comment by jlaura** [comment]
@guymac Sounds good. As @jessemapel indicated, we need additional information in order to address this. I have copied our bug reporting template into my response to assist you in providing the necessary information.

```
**ISIS version(s) affected**: x.y.z

**Description**  
<!-- A clear and concise description of the problem. -->

**How to reproduce**  
<!-- Code and/or config needed to reproduce the problem. -->

**Possible Solution**  
<!--- Optional: only if you have suggestions on a fix/reason for the bug -->

**Additional context**  
<!-- Optional: any other context about the problem: log messages, screenshots, etc. -->
```

Also, have you tested 3.7.0_RC2?

**Comment by jessemapel** [comment]
@guymac It would be helpful if you could provide a control network file that segfaults along with the command line you called the application with.

Thanks for sticking through the issue migration!

**Comment by guymac** [comment]
It *is* working with Isis 3.7.0; thank you.

**Comment by jlaura** [comment]
@guymac Great to hear. I am going to wager that you have 3.7.0_RC2 (Release Candidate 2) currently? Next week we will do an official release of 3.7.0 so it might be best to pull that down once the release is live.

Are we safe to close this?

**Comment by guymac** [comment]
How do I check? I know it is 3.7.0 from the init script, that's all... I do think this issue can be closed though.

**Comment by jlaura** [comment]
If you ran (which it sounds like you did): `conda install -c usgs-astrogeology/label/RC isis3` you are running the 3.7.0 release candidate. If you just did a conda install isis3, you are on 3.6.2.

I think that a `conda list` in the isis3 environment should list the label as well as the version. I might be wrong on that though.

**Comment by jessemapel** [comment]
When you run `conda list` you should see something like this if you have the release candidate

```
isis3                     3.7.0_RC2                py36_0    usgs-astrogeology/label/RC
```

Regular releases look like this when you run `conda list`

```
isis3                     3.6.2                    py36_0    usgs-astrogeology
```

**Comment by rbeyer** [comment]
Whoa, I'm so glad this was here for me to find.  I was doing some slither testing and ran into this same segfault issue with ISIS 3.6.0.  The problem wasn't with slither, but with hijitreg.  That version of hijitreg created some control network points that slither just barfed on (but not all: I was working with four images, the two IR and the two BG ccds, and it barfed on the IR but not on the BG ccds).  I tried running the 3.8.0 version of slither on control networks created by the 3.6.0 version of hijitreg, and they bombed, but if I backed up a step, and ran the 3.8.0 hijitreg, then the 3.8.0 slither worked just fine.

This shouldn't be an issue unless someone is still working with or testing this with ISIS 3.6.0, but I just figured I'd leave this here in case it is of use.
